### PR TITLE
[ADD] iot_output

### PR DESCRIPTION
**Pull request by etobella**
This module allows different outputs for the same device.

### CODE REVIEW

**Review by jarroyomorales — APPROVED** [review]
LGTM

File: iot_output/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2018 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'IoT Output',
+    'version': '11.0.1.0.1',
```
Comment (LoisRForgeFlow) [inline iot_output/__manifest__.py]: first version in oca so 11.0.1.0.0

File: iot_output/security/ir.model.access.csv [hunk 2]
```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_iot_device_output_action,access_iot_device_output_action,model_iot_device_output_action,iot.group_iot_user,1,1,1,0
+manage_iot_device_output_action,access_iot_device_output_action,model_iot_device_output_action,iot.group_iot_manager,1,1,1,0
```
Comment (LoisRForgeFlow) [inline iot_output/security/ir.model.access.csv]: same rights for user and manager?

**Review by LoisRForgeFlow — COMMENTED** [review]
Code review

**Comment by etobella** [comment]
@lreficent Fixed!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by etobella** [comment]
The issue was related to #8

**Pull request merged**